.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -78,13 +78,22 @@ body:
 
         You can attach images or log files by clicking this area to highlight it and then dragging files in.
         If GitHub upload is not working, you can also copy and paste the output into this section.
-  - type: checkboxes
-    id: terms
+  - type: dropdown
+    id: code_of_conduct
     attributes:
       label: Code of Conduct
-      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/catalystneuro/roiextractors/blob/master/.github/CODE_OF_CONDUCT.rst)
+      description: Do you agree to follow our [Code of Conduct](https://github.com/catalystneuro/roiextractors/blob/master/.github/CODE_OF_CONDUCT.rst)?
+      options:
+        - "Yes"
+    validations:
+      required: true
+  - type: dropdown
+    id: redundant_issue_check
+    attributes:
+      label: Redundant Issue Check
```

```suggestion
    id: duplicated_issue_check
    attributes:
      label: Duplicated Issue Check
```

'Redundant' might be a bit strong; that qualifies the issue as not mattering but in reality it matters, it's just already open and in progress in another ticket